keyboards/monsgeek/m2/readme.md

```diff
@@ -0,0 +1,25 @@
+# MonsGeek M2
+[![M2](https://s21.ax1x.com/2024/03/23/pFhDAqH.md.png)](https://imgse.com/i/pFhDAqH)
+
+A customizable 95% keyboard
+
+* Keyboard Maintainer: [jonylee@hfd](https://github.com/jonylee1986)
+* Hardware Supported: MonsGeek M2 PCB
+* Hardware Availability: [monsgeek](https://www.monsgeek.com/)
```

Comment:
A more accurate link would be: https://www.monsgeek.com/product/m2/

Also, doesn't appear to be wireless.